sample-plugin-candlestick-chart-new/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-stockcharts-example2",
+  "description": "Examples for react-stockcharts",
+  "version": "0.0.24",
+  "dependencies": {
+    "@sigmacomputing/plugin": "^1.0.5",
+    "d3-dsv": "^1.0.8",
+    "d3-format": "^1.2.1",
+    "d3-scale": "^1.0.7",
+    "d3-time-format": "^2.1.1",
+    "prop-types": "^15.6.0",
+    "react": "^16.2.0",
```

Comment:
You should be using at least react 16.9 at minimum. We use react `17.0.2` in slate but you should probably use react `18`. Also update `react-dom` to the same version